src/react/components/pages/train/trainPage.scss

```diff
@@ -41,6 +39,11 @@
         &:last-child {
             border-bottom: #999 2px solid;
         }
+        .tag-name {
```

Comment:
what is 550 derived from?

Reply:
It's widths of right pane, and left bar , plus some space between.